homestar-runtime/src/settings.rs

```diff
@@ -133,3 +187,16 @@ impl Settings {
         s.try_deserialize()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::Settings;
+
+    #[test]
+    fn load_settings() {
+        let settings = Settings::load().unwrap();
+
+        println!("{:?}", settings.node.network.keypair_config.generate_keypair())
```

Comment:
minor: guessing we'll have a real assert here, so just mentioning it aloud.

Reply:
actual tests to be a thing are still in the works